test/Multichain.js

```diff
@@ -4,116 +4,131 @@ let expect = chai.expect
 var Multichain = require("../build/transport/Multichain").Multichain
 var MultichainConnection = require("../build/transport/Multichain").MultichainConnection
 var path = require("path")
+const mock = require('./multichain-data/multichainMock.json')
 require("dotenv").config({ path: path.join(__dirname, "..", "build", "test.env") })
 
 describe('Multichain', () => {
+    var multichain
+    before(function () {
+        multichain = makeConnectedMultichainObject()
+    })
+
     it('connects to a valid multichain instance', () => {
-        var multichain = makeConnectedMultichainObject()
-        multichain.Info(function (err, info) {
-            expect(err).to.not.exist
+        multichain.Info(function (error, info) {
```

Comment:
The only code under test invoked in this test is 'makeConnectedMultiChainObject' and 'Info'.  Info is an accessor; makeConnnected... is a factory method that invokes the constructor with 2 args. Assertion about the version seems like it should be a test of the constructor; this test should focus on what's done in the method under test.